src/test/java/uk/gov/homeoffice/digital/sas/person/CallistoPersonRestApiApplicationTests.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.homeoffice.digital.sas.person;
+
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+class CallistoPersonRestApiApplicationTests {
```

Comment:
nit: really minor, I noticed all our test classes in timecard are named with 'Test' singular without the 's'

Reply:
Agreed. This was a copy and paste side effect 😄 . Fixed

Reply:
I can't actually see the change, but will resolve anyway as it's just nit picking

Reply:
My mistake. I forgot to push the change. It should be now